samples/dnn/person_reid.cpp

```diff
@@ -0,0 +1,255 @@
+//  This file is part of OpenCV project.
+//  It is subject to the license terms in the LICENSE file found in the top-level directory
+//  of this distribution and at http://opencv.org/license.html.
+// 
+// 
+//                           License Agreement
+//                 For Open Source Computer Vision Library
```

Comment:
OpenCV samples don't use license headers.
These files are used as a part of documentation which may use whole sample's file or its snippets.

You may left here short notice about sample's authors and/or model's authors (and/or model's limitations).

https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure